rasterio/tool.py

```diff
@@ -48,6 +49,71 @@ def stats(source):
     return Stats(numpy.min(arr), numpy.max(arr), numpy.mean(arr))
 
 
+def show_hist(source, bins=10, masked=True, title='Histogram'):
```

Comment:
@geowurster I want to retract that previous request this morning. Automatically putting the dataset name in the title doesn't make sense for `show_hist()`.